src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -528,6 +554,16 @@ private void validateOrderingKeys(final Schema schema, final EventTypeBase event
         }
     }
 
+    private void validatePartitionCompactionKeys(final Schema schema, final EventTypeBase eventType)
+            throws InvalidEventTypeException, JSONException, SchemaException {
+        final List<String> absentFields = eventType.getPartitionCompactionKeys().stream()
+                .filter(field -> !schema.definesProperty(convertToJSONPointer(field)))
```

Comment:
@rcillo I didn't find a fast way to validate that there is a required field exists in schema. So here I only validate that property exists in schema (we actually do the same for other ..._key_fields properties). If you have an idea regarding that - please let me know and I will change it.